changelog.md

```diff
@@ -21,6 +21,11 @@
 - VerneMQ now requires Erlang/OTP 21.2 or newer to build.
 - Use `atomics` and `persistent_term` to handle metrics instead of using the
   `mzmetrics` library which has been removed.
+- Change `vmq_reg:publish/4` to return `{ok, {NumLocalMatches, NumRemoteMatchs}}`
+  instead of `ok`. This introduces a breaking change to the *inofficial* publish
```

Comment:
Although the Merriam-Webster’s dictionary mentions `inofficial` as a synonym, `unofficial` is the most common method that English speaking lads use.